Library/Formula/dfmt.rb

```diff
@@ -4,6 +4,10 @@ class Dfmt < Formula
   url "https://github.com/Hackerpilot/dfmt.git",
       :tag => "v0.4.4",
       :revision => "c53eb5e0e4a6bb494dccc1eb1ff3e6f4dc820bfa"
+  
+  devel "https://github.com/Hackerpilot/dfmt.git",
```

Comment:
Needs to be `devel do`, not just `devel`. Should be beneath the `bottle do` block as well :wink: